src/ontodia/diagram/model.ts

```diff
@@ -41,9 +41,10 @@ export class DiagramModel extends Backbone.Model {
     elements: { [id: string]: Element } = {};
     linksByType: { [type: string]: Link[] } = {};
 
-    constructor(isViewOnly = false) {
+    constructor(isViewOnly = false, linkStyles?: {link?: Object; label?: Object; router?: string; connector?: string}) {
```

Comment:
Why `linkStyles` (an options for **view**) is being passed to `DiagramModel` (a **model** of diagram)?